code/modules/mob/living/carbon/human/species/station/alraune.dm

```diff
@@ -0,0 +1,336 @@
+		/mob/living/carbon/human/proc/succubus_drain_lethal,
+		/mob/living/carbon/human/proc/bloodsuck) //Give them the voremodes related to wrapping people in vines and sapping their fluids
+
+	color_mult = 1
+	icobase = 'icons/mob/human_races/r_human_vr.dmi'
+	deform = 'icons/mob/human_races/r_def_human_vr.dmi'
+	flesh_color = "#9ee02c"
+	blood_color = "#edf4d0" //sap!
+	base_color = "#1a5600"
+
+	blurb = "Alraunes are a rare sight in space. Their bodies are reminiscent of that of plants, and yet they share many\
+	traits with other humanoid beings.\
+	\
+	Most Alraunes are not interested in traversing space, their heavy preference for natural environments and general\
+	disinterest in things outside it keeps them as a species at a rather primal stage.\
+	\
+	However, after their discovery by the angels of Sanctum, many alraunes succumbed to their curiosity, and took the offer\
+	to learn of the world and venture out, whether it's to Sanctum, or elsewhere in the galaxy."
+
+	has_limbs = list(
+		BP_TORSO =  list("path" = /obj/item/organ/external/chest),
+		BP_GROIN =  list("path" = /obj/item/organ/external/groin),
+		BP_HEAD =   list("path" = /obj/item/organ/external/head),
+		BP_L_ARM =  list("path" = /obj/item/organ/external/arm),
+		BP_R_ARM =  list("path" = /obj/item/organ/external/arm/right),
+		BP_L_LEG =  list("path" = /obj/item/organ/external/leg),
+		BP_R_LEG =  list("path" = /obj/item/organ/external/leg/right),
+		BP_L_HAND = list("path" = /obj/item/organ/external/hand),
+		BP_R_HAND = list("path" = /obj/item/organ/external/hand/right),
+		BP_L_FOOT = list("path" = /obj/item/organ/external/foot),
+		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right)
+		)
+
+	// limited organs, 'cause they're simple
+	has_organ = list(
+		O_LIVER =    /obj/item/organ/internal/liver/alraune,
+		O_KIDNEYS =  /obj/item/organ/internal/kidneys/alraune,
+		O_BRAIN =    /obj/item/organ/internal/brain/alraune,
+		O_EYES =     /obj/item/organ/internal/eyes/alraune,
+		)
+
+/datum/species/alraune/can_breathe_water()
+	return TRUE //eh, why not? Aquatic plants are a thing.
+
+
+/datum/species/alraune/handle_environment_special(var/mob/living/carbon/human/H)
+	if(H.inStasisNow()) // if they're in stasis, they won't need this stuff.
+		return
+
+	//setting these here 'cause ugh the defines for life are in the wrong place to compile properly
+	//set them back to HUMAN_MAX_OXYLOSS if we move the life defines to the defines folder at any point
+	var/ALRAUNE_MAX_OXYLOSS = 1 //Defines how much oxyloss humans can get per tick. A tile with no air at all (such as space) applies this value, otherwise it's a percentage of it.
+	var/ALRAUNE_CRIT_MAX_OXYLOSS = ( 2.0 / 6) //The amount of damage you'll get when in critical condition. We want this to be a 5 minute deal = 300s. There are 50HP to get through, so (1/6)*last_tick_duration per second. Breaths however only happen every 4 ticks. last_tick_duration = ~2.0 on average
```

Comment:
Why are these in allcaps and not defines?

Reply:
For some reason it refused to compile when I pointed them at the HUMAN_MAX_OXYLOSS define. No idea _why_ it can't find them when it's further down the tree from where they're defined.

Reply:
They may be #undef'd at the bottom of their file or somewhere earlier.

Reply:
Aha, I might have a look at that when I get out of work.